src/App.css

```diff
@@ -1,22 +1,23 @@
 .App {
   text-align: center;
 }
-
-.App-logo {
-  height: 40vmin;
+#main-links > a {
+  margin: 10px;
+  color: #34495e;
+  text-shadow: -1px -1px 0 #ecf0f1, 1px -1px 0 #ecf0f1, -1px 1px 0 #ecf0f1,
+    1px 1px 0 #ecf0f1;
 }
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+ img {
+  height: 10vw;
+  width: 10vw
 }
 
-.App-link {
-  color: #09d3ac;
+span {
+  height: 10vw;
+  width: 10vw;
+  margin: 20px;
+  color: random;
+}
+.movies-con{
```

Comment:
don't be stingy with names. say `movies-container` instead of `con` - it's clearer and doesn't cost you much